joint_calling/upload_processor.py

```diff
@@ -67,11 +78,27 @@ def batch_move_files(
         " """
 
     jobs = []
-    for sample in files:
-        previous_location = os.path.join('gs://', source_prefix, sample)
-        new_location = os.path.join('gs://', destination_prefix, sample)
 
-        j = batch.new_job(name=f'move {sample}')
+    external_id = sample_group.sample_id_external
+    internal_id = sample_group.sample_id_internal
+    batch_number = sample_group.batch_number
+
+    files_to_move = (sample_group.data_file, sample_group.index_file, sample_group.md5)
+
+    for file_name in files_to_move:
+        base = file_name.basename
+        previous_location = file_name.path
+
+        # previous_location = os.path.join('gs://', source_prefix, file_name)
+        # previous_location = file_name
+
+        file_extension = base[len(external_id) :]
```

Comment:
Make sure you import os.path at the top of the file.

```suggestion
        file_extension = os.path.splitext(base)[1]
```